iot-e2e-tests/pom.xml

```diff
@@ -1,116 +1,34 @@
-<!--
-  ~  Copyright (c) Microsoft. All rights reserved.
-  ~  Licensed under the MIT license. See LICENSE file in the project root for full license information.
-  --><!-- Licensed under the MIT license. See LICENSE file in the project root for full license information. -->
+<!-- Copyright (c) Microsoft. All rights reserved. -->
+<!-- Licensed under the MIT license. See LICENSE file in the project root for full license information. -->
 <project>
     <modelVersion>4.0.0</modelVersion>
     <groupId>com.microsoft.azure.sdk.iot</groupId>
-    <artifactId>iot-e2e-tests</artifactId>
-    <name>IoT Hub Java SDK tests</name>
-    <version>0.10.13</version>
-    <description>Test suite fot the Microsoft Azure IoT Device SDK for Java</description>
+	<artifactId>iot-e2e-tests</artifactId>
+	<name>IoT Hub Java SDK tests</name>
+	<version>0.9.13</version>
+	<description>Test suite fot the Microsoft Azure IoT Device SDK for Java</description> 
```

Comment:
Please make sure all the versions correctly reflect what is present on master branch